xml/System/String.xml

```diff
@@ -13296,13 +13296,13 @@ The following example defines a `StripStartTags` method that uses the <xref:Syst
  :::code language="fsharp" source="~/snippets/fsharp/System/String/Substring/source.fs" id="Snippet1":::
  :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR_Classic/classic String.Substring1 Example/VB/source.vb" id="Snippet1":::
 
- The following example uses the <xref:System.String.Substring%2A> method to separate key/value pairs that are delimited by an equals ("=") character.
+ The following example uses the <xref:System.String.Substring%2A> method to separate key/value pairs that are delimited by an equals (`"="`) character.
 
  :::code language="csharp" source="~/snippets/csharp/System/String/Substring/Substring1.cs" interactive="try-dotnet-method" id="Snippet1":::
  :::code language="fsharp" source="~/snippets/fsharp/System/String/Substring/Substring1.fs" id="Snippet1":::
  :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_CLR_System/System.String.Substring/vb/Substring1.vb" id="Snippet1":::
 
- The <xref:System.String.IndexOf%2A> method is used to get the position of the equals character in the string. The call to the <xref:System.String.Substring(System.Int32%2CSystem.Int32)> method extracts the key name, which starts from the first character in the string and extends for the number of characters returned by the call to the <xref:System.String.IndexOf%2A> method. The call to the <xref:System.String.Substring(System.Int32)> method then extracts the value assigned to the key. It starts at one character position beyond the equals character and extends to the end of the string.
+ The <xref:System.String.IndexOf%2A> method is used to get the position of the equals character in the string. The call to the <xref:System.String.Substring(System.Int32%2CSystem.Int32)> method extracts the key name, which starts from the first character in the string and extends for the number of characters returned by the call to the <xref:System.String.IndexOf%2A> method. The call to the <xref:System.String.Substring(System.Int32)> method then extracts the value assigned to the value. It starts at one character position beyond the equals character and extends to the end of the string.
```

Comment:
I think this was correct as initially written - the key (before the =) has a value (after the =).

Reply:
Oh, you’re right. I must have missed the part where it said _“extracts **the value assigned** to the key”_. I’ll revert that.